Dockerfile

```diff
@@ -1,29 +1,32 @@
 FROM openjdk:8-jdk
```

Comment:
{suggestion} Look at/use the official maven images. Building from them (iirc) can handle the maven wget & copy+build. Could mod the entrypoint to run from target to make the docker file smaller. (Personally I'd have a separate build and a run setup)

Reply:
Sounds good